### DISCUSSION TITLE
How many computer units does the notebook cost per hour?

### DISCUSSION THREAD

**Discussion by Nojarosann** [TheLastBen/fast-stable-diffusion#2261] [Q&A]
I don't have a credit card to get the other options but google has an option to pay it without using a credit card(At least here where i live) So i wanted to know how much the notebook uses per hour mostly so i can have an idea how much i'm gonna spend with it

**Comment by TheLastBen** [comment]
100 unites will get you 50 hours of T4

**Reply by Nojarosann** [reply]
i assume T4 is a GPU series? It's just that i've seen people talking about it costing between 1.9 to 1.8 per hour and that it has been going down or something? I was wondering if you knew how much it is costing for an average 512x512 image. i'm sorry for the inconvenience

**Reply by TheLastBen** [reply]
like, you said, it's 1.9 per hour
